apps/02RockPaperScissors.js

```diff
@@ -11,7 +11,43 @@ var rl = readline.createInterface({
 function rockPaperScissors(hand1, hand2) {
 
   // Write code here
+  if (hand1.toLowerCase() === hand2.toLowerCase()) {
+  return 'It\'s a tie!';
+  }
 
+  if (hand1.toLowerCase() === 'rock'){
+
+     if (hand2.toLowerCase() === 'scissors') {
+          return 'Hand one wins!';
+        }
+
+   
+        // If we reach here, player 2 must have dealt paper
+          return 'Hand two wins!';
```

Comment:
if you don't have a conditional statement the app will always stop here. Clear this out or add a statement around it.